lib/standalone.js

```diff
@@ -174,11 +175,9 @@ module.exports = function (options) {
 		const cwd = (globbyOptions && globbyOptions.cwd) || process.cwd();
 		const absolutePath = !path.isAbsolute(entry) ? path.join(cwd, entry) : path.normalize(entry);
 
-		entry = entry.replace(/\\(?![[\]()])/g, '/');
-
 		if (fs.existsSync(absolutePath)) {
 			// This path points to a file. Return an escaped path to avoid globbing
-			return fastGlob.escapePath(entry);
+			return fastGlob.escapePath(normalizePath(entry));
```

Comment:
This only works when path is absolute, if applying a pattern like "**/*.scss" that is not absolute, it still fails.
I tried to run with 
entry = entry.replace(/\\(?![[\]()])/g, '/');

it works, I have not read all comments what was the reason to remove it.

Reply:
entry = normalizePath(entry) needs to be before if (fs.existsSync(absolutePath)) {

Reply:
@VikasAgarwal1984  this is an old (merged) pr, so if you think something needs changes, you should create a new pr for the changes.